docs/release-notes/index.md

```diff
@@ -0,0 +1,163 @@
+---
+navigation_title: EDOT Node.js
+description: Release notes for Elastic Distribution of OpenTelemetry Node.js.
+applies_to:
+  stack:
+  serverless:
+    observability:
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: edot-sdk
+---
+
+# Elastic Distribution of OpenTelemetry Node.js release notes [edot-nodejs-release-notes]
+
+Review the changes, fixes, and more in each version of Elastic Distribution of OpenTelemetry Node.js.
+
+To check for security updates, go to [Security announcements for the Elastic stack](https://discuss.elastic.co/c/announcements/security-announcements/31).
+
```

Comment:
Would it be reasonable to add a link to the breaking changes doc either here at the top, or in each version section? One of the major cons to the new splitting of release notes and breaking changes, IMO, is having two separate places to look for "what's changed in this new version". Adding a link from here to possible breaking changes would alleviate that a little.

Reply:
Sure!